sphinxcontrib/devhelp/__init__.py

```diff
@@ -78,11 +76,17 @@ def build_devhelp(self, outdir, outname):
         logger.info(__('dumping devhelp index...'))
 
         # Basic info
+
+        # TODO: author attrib
+        # TODO: language attrib
```

Comment:
How about using `config.language`? Is it needed to normalize?

Reply:
Language is about the programming language. Could this be config.laguage? The relationship between the contents of the [documentation](https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-internationalization) attribute value is different.

Reply:
@tk0miya This attribute is similar to this one¹ (language: python) found in Travis-CI configurations. I have not so full domain of Sphinx, but with some research in the documentation, I have not found anything like that readable from "config.py".

I thought of two possibilities. The first one is to leave the value of the parameter attribute as "unknown" or make an analysis in the existing domains and the value of the attribute can be the predominant domain (eg Python or C). Understood? What do you think about this?